src/pages/Schedule.jsx

```diff
@@ -0,0 +1,101 @@
+import styled from 'styled-components';
+import theme from '../styles/theme';
+import { timeList, careTimeList } from '../constants/staticData';
+
+const Schedule = () => {
+  return (
+    <ContainerStyled>
+      <h2>돌봄 스케줄을 설정해주세요</h2>
+      <InputWrapStyled>
+        <CalendarWrapStyled>
+          <div>시작일</div>
+          <CalendarButton>
+            <span>월 일</span>
+            <span>아이콘</span>
+          </CalendarButton>
+        </CalendarWrapStyled>
+        <CalendarWrapStyled>
+          <div>종료일</div>
+          <CalendarButton>
+            <span>월 일</span>
+            <span>아이콘</span>
+          </CalendarButton>
+        </CalendarWrapStyled>
+      </InputWrapStyled>
+      <SelectWrapStyled>
+        <div>돌봄 시작 시간</div>
+        <select name="돌봄 시작 시간">
+          {timeList.map((el, idx) => {
+            return <option key={idx}>{el.text}</option>;
+          })}
+        </select>
+      </SelectWrapStyled>
+      <SelectWrapStyled>
+        <div>하루 돌봄 시간</div>
+        <select name="" id="">
+          {careTimeList.map((el, idx) => {
+            return <option key={idx}>{el.text}</option>;
+          })}
```

Comment:
화살표 함수에서 곧장 반환하는 경우 return이 필요 없습니다.
```suggestion
          {careTimeList.map((el, idx) => <option key={idx}>{el.text}</option>)}
```
이렇게 더 깔끔하게 작성할 수 있습니다.